airbyte-ci/connectors/metadata_service/orchestrator/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "orchestrator"
-version = "0.1.4"
+version = "0.1.5"
```

Comment:
Nit: That's prob a 0.2 at least in my book.

Reply:
I consider it's a fix of something which was working before, this is why I made a patch bump.